cli/tests/testdata/run/tls_connecttls.js

```diff
@@ -9,7 +8,7 @@ import { TextProtoReader } from "./textproto.ts";
 const encoder = new TextEncoder();
 const decoder = new TextDecoder();
 
-const resolvable = deferred();
+const { promise, resolve } = Promise.withResolvers();
```

Comment:
Maybe let's not destructure bindings here and name it `resolveable`?

Reply:
No strong opinions against this but how come?

Reply:
In some tests you'll need to create multiple deferreds, I think it's a bit easier to understand if you have `resolvable1` and `resolvable2` than `promise1`, `resolve1`, reject1`, `promise2`, `resolve2`, `reject2`.

Reply:
I see. I've only destructured in tests that contain 1 occurrence of `Promise.withResolvers()`. The rest adhere to your suggestion.